src/relay.js

```diff
@@ -0,0 +1,30 @@
+import {fromGlobalId, connectionFromArray, nodeDefinitions} from 'graphql-relay';
+
+export function idFetcher(sequelize) {
+  return globalId => {
+    const {type, id} = fromGlobalId(globalId);
+    const models = sequelize.models.map(model => model.name);
```

Comment:
Working through some of these bugs while i'm testing the node queries. will submit some changes shortly .